src/Transform.jl

```diff
@@ -40,17 +33,14 @@ function logit{NumType <: Number}(x::NumType)
 end
 
 
-function logit{NumType <: Number}(x::Array{NumType})
-    1.0 ./ (1.0 + exp(-x))
-end
-
-
 """
 Convert an (n - 1)-vector of real numbers to an n-vector on the simplex, where
 the last entry implicitly has the untransformed value 1.
 """
 function constrain_to_simplex{NumType <: Number}(x::Vector{NumType})
     if any(x .== Inf)
+        # Is there a bug here? If more than 1 entry in x is Inf, that just means
+        # that the last entry in z is 0, not the both entries in z are the same.
```

Comment:
Since infinity is unordered, the only other solution is to throw an error.  Since you can't always control what an optimization algorithm does, that's not very desirable.  What do you think would be better?